src/Adjudications/AdjudicationHistory/AdjudicationHistoryFormValidation.js

```diff
@@ -0,0 +1,21 @@
+import { FORM_ERROR } from 'final-form'
+
+const validateThenSubmit = ({ fromDate, toDate, submit }) => values => {
+  if (toDate && fromDate && fromDate.isAfter(toDate)) {
```

Comment:
will you be adding the other validation at a later date e.g. dates are required, `toDate` must be in the past etc...

Reply:
as discussed - other validation isn't required as its click based so you can't put in invalid fields